operations/helm/charts/mimir-distributed/README.md.gotmpl

```diff
@@ -60,6 +60,8 @@ As part of this chart many different pods and services are installed which all
 have varying resource requirements. Please make sure that you have sufficient
 resources (CPU/memory) available in your cluster before installing Grafana Mimir Helm Chart.
 
+For a more detailed guide on setting up Grafana Mimir refer to [Getting started with Grafana Mimir using the Helm chart](https://grafana.com/docs/mimir/latest/operators-guide/deploying-grafana-mimir/getting-started-helm-charts/).
```

Comment:
```suggestion
For details about how to set up Grafana Mimir, refer to [Get started with Grafana Mimir using the Helm chart](https://grafana.com/docs/mimir/latest/operators-guide/deploying-grafana-mimir/get-started-helm-charts/).
```

Reply:
but the article is called getting started with the helm chart...... which one is it - the name of the article or the correct way of saying things